_config.yml

```diff
@@ -150,3 +147,12 @@ exclude:
   - package-lock.json
   - README.md
   - src
+  - CONTRIBUTING.md
+  - LICENSE.md
+  - assets
```

Comment:
Only copy over files that are necessary since EVERYTHING ends up on S3. In particular, we no longer need to copy over `assets` since `jekyll-assets` is doing to work for us.